smoke-tests/designer/features/pageobjects/pages/addComponent.page.js

```diff
@@ -34,8 +34,8 @@ class AddComponentPage extends Page {
   }
 
   selectComponentByName(name) {
-    this.selectList.waitForDisplayed();
-    this.selectList.selectByVisibleText(name);
+    this.addComponent.waitForDisplayed();
+    browser.$(`=${name}`).click();
```

Comment:
There's a `selectByName` method for this?

http://v4.webdriver.io/v3.4/api/action/selectByName.html

Reply:
@jonnywyatt we're no longer using a select list as it's not very accessible

Reply:
👍